### PR TITLE
[ticket/13414] Set Content-Length header only when 304 status is not sen...

### PR DESCRIPTION
**Pull request by marc1706**
Ticket: https://tracker.phpbb.com/browse/PHPBB3-13414

### CODE REVIEW

File: phpBB/includes/functions_download.php [hunk 1]
```diff
@@ -210,16 +210,16 @@ function send_file_to_browser($attachment, $upload_dir, $category)
...
-	{
-		header("Content-Length: $size");
-	}
-
 	// Close the db connection before sending the file etc.
 	file_gc(false);
 
 	if (!set_modified_headers($attachment['filetime'], $user->browser))
 	{
+		if ($size)
```
Comment (bantu) [inline phpBB/includes/functions_download.php]: Move this block below Sendfile handling?

**Comment by bantu** [comment]
I'd generally like to have this in 3.0.x too.

**Comment by marc1706** [comment]
@bantu send another PR based on develop-olympus then?

File: phpBB/includes/functions_download.php (hunk 1 above)
Comment (marc1706) [inline phpBB/includes/functions_download.php]: sure.

**Comment by marc1706** [comment]
Closing in favor of https://github.com/phpbb/phpbb/pull/3343

**Pull request closed without merging**